kubernetes/samples/scripts/create-rcu-schema/README.md

```diff
@@ -1,117 +1,108 @@
 # Creating Oracle DB service and RCU Schema for a Fusion Middleware domain
 
-This sample demonstrates how to create an Oracle DB service on Kubernetes cluster and create RCU schema on the Oracle DB being used by a Fusion Middleware domain.  
-
-The directory contains the following sample scripts to start an Oracle DB service in default namespace, stop the Oracle DB service, create an RCU schema and drop the RCU schema
-
-```
-$ ./start-db-service.sh   
-Trying to pull repository container-registry.oracle.com/database/enterprise ... 
-12.2.0.1-slim: Pulling from container-registry.oracle.com/database/enterprise
-Digest: sha256:25b0ec7cc3987f86b1e754fc214e7f06761c57bc11910d4be87b0d42ee12d254
-Status: Image is up to date for container-registry.oracle.com/database/enterprise:12.2.0.1-slim
+The sample scripts in this directory demonstrates how to:
+* Start an Oracle DB service in a Kubernetes cluster.
+* Stop an Oracle DB service in a Kubernetes cluster.
+* Create the RCU schema in the Oracle DB that will be used by a Fusion Middleware domain.
+* Delete the RCU schema in the Oracle DB used by a Fusion Middleware domain.
+
+## Start an Oracle DB service in a Kubernetes cluster
+```
+The script creates an Oracle DB Service in default namespace with default credential that comes with the Oracle Database Slim image.
+
+The script assumes either the image container-registry.oracle.com/database/enterprise:12.2.0.1-slim is available in docker image repository or an ImagePullSecret is created for container-registry.oracle.com. See the script create-image-pull-secret.sh to create secret on container-registry.oracle.com .
```

Comment:
"available in docker image repository" -> "available in the Docker repository"
"to create secret on" -> "to create a secret for accessing"